src/d3a/models/strategy/external_strategies/storage.py

```diff
@@ -73,9 +73,10 @@ def _delete_offer(self, payload):
 
     def _delete_offer_impl(self, arguments, response_channel):
         try:
+            self.state.offered_sell_kWh[self.market.time_slot] -= \
```

Comment:
Outside of the context of this story, but we had a discussion with @hannesdiedrich in his PR that it would be better to recalculate the offered_sell_kWh from the energy of the posted offers after the offer was deleted. I think that he is dealing with it in his PR though, correct me if I am wrong.

Reply:
Yes, I am deeling with this here: https://github.com/gridsingularity/d3a/pull/666

Reply:
LGTM then :)